integration/containerd/confidential/agent_image.bats

```diff
@@ -93,7 +93,6 @@ setup() {
 
 	echo "Reconfigure Kata Containers"
 	switch_image_service_offload on
-	clear_kernel_params
```

Comment:
My intention with `clear_kernel_params` is to avoid that kernel parameters added by a test, interfere in the results of the next tests. When I implemented this, `kernel_params` was always empty at the begin of the tests so it made sense to simply clean up everything on `teardown()`. Continue calling `clear_kernel_params` became wrong, but removing it is also not entirely correct; we should find a better solution...

... I can think in two solutions, both based on save-and-restore approach:
1. save the entire kata's configuration.toml on `setup()` and restore on `teardown()`
2. likewise but save/store only the `kernel_params` value;

Do you see any other solution?

Reply:
I updated the PR with `kernel_params` save and restore.